migrations/versions/b984311d26d7_add_modified_by_created_by_fields_for_.py

```diff
@@ -0,0 +1,81 @@
+"""Add modified_by/created_by fields for traceability.
+
+Revision ID: b984311d26d7
+Revises: 072487c42d98
+Create Date: 2017-11-21 15:50:10.745763
+
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import sqlalchemy as sa
+from alembic import op
+
+# Revision identifiers, used by Alembic.
+revision = 'b984311d26d7'
+down_revision = '072487c42d98'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    """Add 'modified_by'/'created_by' cols and backfill clients/collections/users/permissions."""
```

Comment:
Nitpick: this doesn't perform the backfill now, right?

Reply:
Will fix!